it/controllers/HomeControllerISpec.scala

```diff
@@ -30,54 +30,26 @@ class HomeControllerISpec extends ComponentSpecBase with GenericStubMethods with
 
   "Navigating to /report-quarterly/income-and-expenses/view" when {
 
-    "Authorised and" when {
+    "Authorised" should {
 
-      "the home page feature is enabled" should {
+      "render the home page" in {
 
-        "render the home page" in {
-          appConfig.features.homePageEnabled(true)
-
-          isAuthorisedUser(true)
-          stubUserDetails()
-
-          When("I call GET /report-quarterly/income-and-expenses/view")
-          val res = IncomeTaxViewChangeFrontend.getHome
-
-          Then("the result should have a HTTP status of OK (200) and the Income Tax home page")
-          res should have(
-
-            //Check Status OK (200) Result
-            httpStatus(OK),
-
-            //Check Redirect Location
-            pageTitle("Your Income Tax")
-          )
-        }
-      }
```

Comment:
Keep this test in - just without the appConfig.enabled bit

Reply:
I think it's just not being shown properly for some reason, if you go onto the file I think it should be okay.